content/200-concepts/100-components/01-prisma-schema/04-data-model.mdx

```diff
@@ -882,6 +882,71 @@ enum Role {
 </tab>
 </TabbedContent>
 
+## Defining composite types
+
+<Admonition type="info">
+
+Composite types are available in Preview in versions `3.10.0` and later.
+
+</Admonition>
+
+<Admonition type="warning">
+
+Composite types are currently only available on MongoDB.
+
+</Admonition>
```

Comment:
Would we like to merge the 2 sections into one? 🤔

Reply:
It's two different ones - an info and a warning. Different color in rendering I think.